x-pack/plugins/security_solution/public/common/components/event_details/event_fields_browser.tsx

```diff
@@ -7,6 +7,8 @@
 import { sortBy } from 'lodash';
 import { EuiInMemoryTable } from '@elastic/eui';
 import React, { useCallback, useMemo } from 'react';
+
+import { getOr } from 'lodash/fp';
```

Comment:
can we fix `import { sortBy } from 'lodash';` to import from `lodash/fp` and have those imports merged together?